src/linker/runner.py

```diff
@@ -95,14 +97,18 @@ def launch_slurm_job(
     jt.outputPath = f":{str(results_dir / '%A.o%a')}"
     jt.errorPath = f":{str(results_dir / '%A.e%a')}"
     jt.remoteCommand = shutil.which("linker")
-    jt.args = [
+    args = [
         "run-slurm-job",
         container_engine,
         str(results_dir),
         step_name,
         str(step_dir),
         "-vvv",
     ]
+    for filepath in input_data:
+        args.append("--input-data")
+        args.append(f"{str(filepath)}")
```

Comment:
Is there a reason this can't be a single append call in one line?

Reply:
I tried for a second and couldn't figure it out - will look again

Reply:
Hah, this works but is too unreadable imo: `jt_args += [element for filepath in input_data for element in ["--input-data", str(filepath)]]`. I'll comproomise and at least use .extend instead of two .appends

Reply:
Oh I just meant `args.append(f"--input-data {filepath}")`